lib/active_merchant/billing/gateways/paypal_express_common.rb

```diff
@@ -16,13 +16,11 @@ def redirect_url
         test? ? test_redirect_url : live_redirect_url
       end
 
-      def redirect_url_for(token, options = {})
+      def base_redirect_url_for(token, options: {}, cmd: nil)
         options = {:review => true, :mobile => false}.update(options)
```

Comment:
`mobile: false` does have sense when this is `payflow_express` or `digital_goods`. I can change it to `build_redirect_url`.